docs/flowcollector-flows-netobserv-io-v1beta1.adoc

```diff
@@ -136,13 +131,13 @@ Required::
 
 | `ipfix`
 | `object`
-| `ipfix` - _deprecated (*)_ - describes the settings related to the IPFIX-based flow reporter when `spec.agent.type` is set to `IPFIX`.
+| `ipfix` [deprecated (*)] - describes the settings related to the IPFIX-based flow reporter when `spec.agent.type` is set to `IPFIX`.
```

Comment:
@jotak I think this option is no longer deprecated? Since we are supporting in the 1.4 release? I noted this in my PR too, but I think this needs updating. Apologies for missing it before.

Reply:
Be carefull, the `agent.type` **IPFIX** is still deprecated since it's ingestion from OVN flows.

However the one we are supporting on 1.4 is **IPFIX** `export.type`:
https://issues.redhat.com/browse/NETOBSERV-962

The unsupported mention should me removed there:
```
IPFIX configuration, such as the IP address and port to send enriched IPFIX flows to. Unsupported (*).
```
https://github.com/netobserv/network-observability-operator/blob/release-1.4/docs/flowcollector-flows-netobserv-io-v1beta1.adoc#specexporters

https://github.com/netobserv/network-observability-operator/blob/release-1.4/api/v1beta1/flowcollector_types.go#L819-L823